src/controller/bundle.controller.ts

```diff
@@ -97,7 +97,9 @@ export class BundleController extends BaseController {
     });
 
     await streamFinished(bundleStream);
-    res.status(200).json(result).end();
```

Comment:
Maybe we need to go back to simply sending the string - without parsing: send(result)